guides/doc-Planning_Guide/topics/Introduction.adoc

```diff
@@ -27,21 +27,21 @@ ifndef::satellite[]
 include::../common/modules/snip_red-hat-images.adoc[]
 endif::[]
 
-[[figu-Red_Hat_Satellite-Architecture_Guide-Red_Hat_Satellite_6_System_Architecture-Red_Hat_Satellite_6_System_Architecture]]
+[[figu-Architecture_System_Architecture-Red_Hat_Satellite_6_System_Architecture]]
```

Comment:
```suggestion
[[figu-Architecture_System_Architecture-System_Architecture]]
```
We could remove further?

Reply:
Removed (two occurrences) & based to master.